### ISSUE TITLE
Incorrect usage of <dl> element in Step 4?

### ISSUE THREAD

**Issue by jseppi** [18F/calc#939]
The current markup of the description in Step 4 of the Price List Upload flow looks like this:

```html
<dl>
  <li>Term: <dd>value</dd></li>
  ...
</dl>
```

I was just reading over the MDN page for description lists (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) and it looks like the items should instead be formed using sibling `<dt>` (definition term) and `<dd>` elements, like so:

```html
<dl>
  <dt>Term</dt>
  <dd>value<dd>
  ...
</dl>
```

and probably some accompanying css to add a separator after the `<dt>`s:

```css
dt:after {
  content: ": ";
}
```

Thoughts, @hbillings?

**Comment by hbillings** [comment]
Derp. Yes, you're right. >_<

**Comment by jseppi** [comment]
Any fixes for this will also affect `details_list.html` in #927, so this is a reminder to me to merge them in if they become available before #927 is finished.
